BankClient.sln.DotSettings.user

```diff
@@ -1,5 +1,5 @@
 ﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
-	<s:String x:Key="/Default/Environment/Highlighting/HighlightingSourceSnapshotLocation/@EntryValue">C:\Users\Dima\AppData\Local\JetBrains\Rider2022.1\resharper-host\temp\Rider\vAny\CoverageData\_BankClient.123927811\Snapshot\snapshot.utdcvr</s:String>
+	
```

Comment:
Try to use this guide to remove this file only from the repository: https://stackoverflow.com/questions/2047465/how-do-i-delete-a-file-from-a-git-repository

Reply:
Ok